examples/interfaces/nutils_interface.py

```diff
@@ -0,0 +1,130 @@
+from nutils import mesh, function, solver
+from nutils.expression_v2 import Namespace
+
+import gustaf as gus
+import numpy as np
+"""This example shows the funtionality of gustaf.io.nutils.
+
+A 2-dimensional plate with fixed edges on the left and right edges is
+exposed to Gravitation.
+The plate consists of 20 triangular elements and is created with gustaf.
+One can either add the displacements directly to gustaf.vertices or evalute
+the new coordinates and create a new gustaf.mesh.
+"""
+
+
+def main():
+    # Simulation parameters
+    degree = 1
+    btype = "std"
+
+    # Physical Parameters
+    g = 9.81
+    poisson = 0.4
+    rho = 1.2
+    e_mod = 1 * pow(10, 2)
+
+    m = define_mesh()
+
+    # Define Boundaries by Edges
+    m.BC.update({'1': np.array([4, 10]), '2': np.array([49, 55])})
+
+    m_in = m.copy()
+
+    to_nutils = gus.io.nutils.to_nutils_simplex(m)
+    domain, geom = mesh.simplex(**to_nutils)
+
+    # Define the Namespace for Nutils Simulation
+    ns = Namespace()
+    ns.g = g
+    ns.rho = rho
+    ns.x = geom
+
+    ns.define_for('x', gradient='∇', normal='n', jacobians=('dV', 'dS'))
+    ns.ubasis = domain.basis(btype, degree=degree).vector(2)
+    ns.u = function.dotarg('lhs', ns.ubasis)
+    ns.X_i = 'x_i + u_i'
+
+    ns.lmbda = (e_mod * poisson) / (1 + poisson) / (1 - 2 * poisson)
+    ns.mu = e_mod / (1 + poisson)
+
+    ns.strain_ij = '(∇_j(u_i) + ∇_i(u_j)) / 2'
+    ns.energy = 'lmbda strain_ii strain_jj + 2 mu strain_ij strain_ij'
+    ns.energy = 'energy + g rho u_1'
+
+    # Set the boundaries. Note, that bound-keys start with 1,2,3,..
+    sqr = domain.boundary['1'].integral('u_k u_k dS' @ ns, degree=degree * 2)
+    sqr += domain.boundary['2'].integral('u_k u_k dS' @ ns, degree=degree * 2)
+
+    cons = solver.optimize('lhs', sqr, droptol=1e-15)
+
+    energy = domain.integral('energy dV' @ ns, degree=degree * 2)
+
+    lhs = solver.optimize('lhs', energy, constrain=cons)  # displacements
+
+    # With bezier
+    bezier = domain.sample('vertex', 0)
+    coordinates = bezier.eval('X_i' @ ns, lhs=lhs)
+    element_nodes = bezier.tri
+
+    # how many axes?
+    # (we get simplices, so take number of simplex nodes minus 1)
+    n_axes = element_nodes.shape[1] - 1
+
+    # faces or volumes?
+    MeshType = {1: gus.Edges, 2: gus.Faces, 3: gus.Volumes}[n_axes]
+    gustaf_mesh = MeshType(elements=element_nodes, vertices=coordinates)
```

Comment:
```suggestion
    # generate gustaf mesh using information given by nutils
    gustaf_mesh = MeshType(elements=element_nodes, vertices=coordinates)
```
is this what's happening here?